benchmarks/helics/helics_benchmark_util.h

```diff
@@ -95,8 +95,8 @@ inline std::string getCPUModel()
     return std::string{};
 }
 #endif
-/// Generate a report of the compilers used and zmq version linked as well as the version info for HELICS
-/// for use with the benchmarks
+/// Generate a report of the compilers used and zmq version linked as well as the version info for
+/// HELICS for use with the benchmarks
```

Comment:
```suggestion
// Generate a report of the compilers used and zmq version linked as well as the version info for
// HELICS for use with the benchmarks
```

Reply:
I think the /// is a doxygen comment,  probably should change to a /** though.

Reply:
I am going to leave this one and take a look at the orignal code

Reply:
I think you're right, `///` is used for doxygen comments.